pallets/liquidity-pools/src/message.rs

```diff
@@ -415,6 +419,22 @@ pub enum Message {
 		pool_id: u64,
 		currency: u128,
 	},
+	//TODO(cdamian) - do we need to add the deprecated fields?
+	//
+	//
+	// 25 - Freeze tranche tokens
+	// DEPRECATED_Freeze,
+	// 26 - Unfreeze tranche tokens
+	// DEPRECATED_Unfreeze,
+	/// Trigger a redeem request.
+	///
+	/// Note - placeholder that should be updated.
+	TriggerRedeemRequest,
+	/// A multi-directional message used for verifying
+	/// both inbound and outbound messages.
+	MessageProof {
+		proof: [u8; 32],
+	},
```

Comment:
Sorry @cdamian I didn't know, you're feature came with new messages from LPv2. In that case, maybe it makes more sense to add those to lpv2 branch.

Reply:
Maybe this PR can be split into something which is idempotent over how `main` works with LPv1 (it means, not sending any proof, just the original message by one router). And a new very small PR over LPv2 sending the proof.

That way most changes are done over `main` avoiding future git conflicts

Reply:
Yea, I was afraid we might have to do this. Will look into doing a smooth transition.

Reply:
OK, am currently thinking about splitting this in 3 parts, and (roughly) do the following on top of the LPv2 branch:
1 - add the new queue pallet;
2 - use the new queue pallet in the gateway;
3 - add proof processing for inbound/outbound messages;